build.gradle

```diff
@@ -7,11 +7,12 @@ buildscript {
     }
     repositories {
         google()
+        mavenCentral()
```

Comment:
why adding mc?

Reply:
Some artefacts are available on MavenCentral. This lowers the risk of downloading some manipulated artefact from jCenter.

Reply:
yea - but increases the risk of downloading manipulated artifacts from mc - let's keep it as it was.

Reply:
True. But I thought we trust MavenCentral more than jCenter because of its validation mechanism.

Reply:
Feel free to drop that particular commit.